src/lib/core/option/option.spec.ts

```diff
@@ -54,20 +54,6 @@ describe('MatOption component', () => {
         .toBe(0, 'Expected no ripples to show up after click on a disabled option.');
     });
 
-    it('should not show ripples if the ripples are disabled using disableRipple', () => {
```

Comment:
Can you move this test to the select tests rather than removing? e.g. `select.disableRipple = true`.

Seems like a good one to have, given that select only tests the presence of the property.

Reply:
We have a test that checks it already: https://github.com/angular/material2/blob/master/src/lib/select/select.spec.ts#L339